### PR TITLE
MF-3 - Make service ports configurable

### LINKED ISSUE #3: Make service ports configurable
The ports of Mainflux services which can be configured via environment variables should have their corresponding, configurable Helm variable. That variable would need to be propagated in Deployments, Services and Ingress.

CC @anovakovic01

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

Resolves #3

### CODE REVIEW

File: charts/mainflux/templates/adapter_mqtt-deployment.yaml [hunk 1]
```diff
@@ -144,35 +144,35 @@ spec:
...
           stdin: true
           tty: true
           env:
             - name: MF_MQTT_ADAPTER_LOG_LEVEL
               value: {{ default .Values.defaults.logLevel .Values.mqtt.proxy.logLevel }}
-            - name: MF_MQTT_ADAPTER_MQTT_PORT
-              value: "{{ .Values.mqtt.proxy.mqtt_port }}"
-            - name: MF_MQTT_ADAPTER_WS_PORT 
-              value: "{{ .Values.mqtt.proxy.ws_port }}"
+            - name: MF_MQTT_ADAPTER_mqttPort
```
Comment (nmarcetic) [inline charts/mainflux/templates/adapter_mqtt-deployment.yaml:156]: Use the upper case.

Comment (nmarcetic) [reply charts/mainflux/templates/adapter_mqtt-deployment.yaml:156]: Dunno, why you are changing this at all? The same is done in docker-compose, its consistent and more eye-catchy if its WS or MQTT.

Comment (manuio) [reply charts/mainflux/templates/adapter_mqtt-deployment.yaml:156]: Find/Replace :) Fixing this now, thank you!

File: charts/mainflux/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Mainflux Helm Chart
+adapter# Mainflux Helm Chart
```
Comment (drasko) [inline charts/mainflux/README.md:1]: Is this a comment or a title? Looks strange...

**Review by drasko — CHANGES_REQUESTED** [review]

File: charts/mainflux/README.md (hunk 2 above)
Comment (manuio) [reply charts/mainflux/README.md]: Typo

**Review by drasko — APPROVED** [review]
LGTM

**Review by dusanb94 — DISMISSED** [review]

**Review by blokovi — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**